_includes/body/analytics.html

```diff
@@ -0,0 +1,17 @@
+{% if site.google_analytics %}
+  <script>!function (w, d) {
+    /* this file can be removed once GA4 support is added upstream in hydejack */
+    /* See: https://github.com/hydecorp/hydejack/issues/310 */
+    /* This is a variation of the code in PR330 on hydejack */
+
+
+    window.dataLayer = w.dataLayer || [];
+    function gtag() {dataLayer.push(arguments);}
+    gtag('js', new Date());
+
+    /* start tracking, will respect consent if set */
+    gtag('config', '{{ site.google_analytics }}');
+
+    w.loadJSDeferred('https://www.googletagmanager.com/gtag/js?id={{ site.google_analytics }}');
+  }(window, document);</script>
+{% endif %}
```

Comment:
There's a combination of liquid/jekyll code wrapping JS here.  The JS is by and large what Google suggests just with the necessary Jekyll/theme stuff merged in.

![Screen Shot 2023-09-15 at 9 40 44 AM](https://github.com/labzero/guides/assets/1916144/fe281876-c4f9-4aef-8ac6-abd000e8b447)